src/generated_struct_info64.json

```diff
@@ -514,53 +514,53 @@
             "scaleMode": 0
         },
         "EmscriptenGamepadEvent": {
-            "__size__": 1432,
+            "__size__": 1440,
```

Comment:
I don't think this file should be needed as part of this change.  Can you running `./embuilder build sysroot --force` (or `./emcc --clear-cache`) to update your headers?

Reply:
Thanks! worked like a charm.

Reply:
I now realize that my cache and the wasm part of my application was continuing to use structs with `long` whereas the js was attempting to read/write 32-bit ints.